### PR TITLE
update underscore in azure-sb and azure-common

### PR DESCRIPTION
**Pull request by dw511214992**

### CODE REVIEW

**Comment by dw511214992** [comment]
@ramya-rao-a please review it. thanks

File: lib/services/serviceBus/package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
   "dependencies": {
     "azure-common": "^0.9.19",
     "mpns": "2.1.3",
-    "underscore": "^1.8.3",
```
Comment (lirenhe) [inline lib/services/serviceBus/package.json:30]: Will you also increase the version of serviceBus?

Comment (dw511214992) [reply lib/services/serviceBus/package.json:30]: no need, because there is an update which updated it.

**Review by lirenhe — APPROVED** [review]

**Review by jeremymeng — APPROVED** [review]
Looks good to me. Not sure why CI is failing at Copy to Blob step.

**Review by ramya-rao-a — APPROVED** [review]

**Comment by dw511214992** [comment]
Hi @ramya-rao-a  cloud you please merge it? we don't have permission to merge it. thanks

**Comment by dw511214992** [comment]
> Looks good to me. Not sure why CI is failing at Copy to Blob step.

looks like there is something wrong with the authentication of coping to blob

**Comment by ramya-rao-a** [comment]
@dw511214992 Can you please look into the build failure? Am not sure of the consequences of merging the PR with that failure

**Comment by dw511214992** [comment]
> @dw511214992 Can you please look into the build failure? Am not sure of the consequences of merging the PR with that failure

It should caused by the access key of the storage account is not valid. I'm asking who has the permission to access the  storage account.

**Comment by dw511214992** [comment]
@ramya-rao-a it's fixed now. please merge the pr

**Pull request merged**